content/src/view.js

```diff
@@ -735,23 +735,33 @@ function showMiddleButton(which) {
 // SHARE DIALOG
 ///////////////////////////////////////////////////////////////////////////
 
+// TODO(katie): Add share to JS Fiddle option.
 function showShareDialog(opts) {
   if (!opts) {
     opts = { };
   }
 
+  /**
+   * Adds a protocol (http:) to a path if it does not yet have one.
+   * @param {string} path The path to add a protocol to.
+   * @return {string} The path with protocol added if necessary.
+   */
```

Comment:
Hey, let's use human readable comments and use // style for block comments for now.